src/glossary.md

```diff
@@ -41,9 +41,9 @@
 [**Mercurial**]{#mercurial}
 :   A free, cross-platform, distributed source management tool. Source bundles and
     binary packages for Mercurial are available at
-    [http://www.selenic.com/mercurial/wiki/index.cgi](http://www.selenic.com/mercurial/wiki/index.cgi/Mercurial).
+    [https://www.selenic.com/mercurial/wiki/index.cgi](https://www.selenic.com/mercurial/wiki/index.cgi/Mercurial).
     The main Mercurial documentation is available at
-    [http://hgbook.red-bean.com](http://hgbook.red-bean.com/).
+    [https://hgbook.red-bean.com](https://hgbook.red-bean.com/).
```

Comment:
red-bean.com either doesn't support https or has a bad certificate.  Recommend this remain as "http" to avoid broken link.